backend-config/backend-config.go

```diff
@@ -42,11 +42,12 @@ var (
 	LastRegulationSync string
 
 	// DefaultBackendConfig will be initialized be Setup to either a WorkspaceConfig or MultiWorkspaceConfig.
-	DefaultBackendConfig BackendConfig
-	pkgLogger            = logger.NewLogger().Child("backend-config")
-	IoUtil               = sysUtils.NewIoUtil()
-	Diagnostics          diagnostics.DiagnosticsI
-	cacheOverride        cache.Cache
+	DefaultBackendConfig        BackendConfig
+	pkgLogger                   = logger.NewLogger().Child("backend-config")
+	IoUtil                      = sysUtils.NewIoUtil()
+	Diagnostics                 diagnostics.DiagnosticsI
+	cacheOverride               cache.Cache
+	useIncrementalConfigUpdates bool
```

Comment:
benefit to have it be a part of `backendConfigImpl` instead of a global..?

Reply:
I guess for the sake of consistency it makes sense to have it here (all other variables are being declared here)

Reply:
fair enough